xwiki-platform-core/xwiki-platform-repository/xwiki-platform-repository-server-api/src/main/java/org/xwiki/repository/script/RepositoryScriptService.java

```diff
@@ -57,7 +57,7 @@ public class RepositoryScriptService implements ScriptService
 
     /**
      * Store a caught exception in the context, so that it can be later retrieved using {@link #getLastError()}.
-     * 
+     *
```

Comment:
My IDE really wants to clean up those spaces. I'll move them out of the PR when ready.